packages/contracts/hardhat.config.ts

```diff
@@ -3,6 +3,13 @@ import "@nomicfoundation/hardhat-toolbox";
 
 const config: HardhatUserConfig = {
   solidity: "0.8.18",
+  networks: {
+    hardhat: {},
+    optimismGoerli: {
+      url: `https://optimism-goerli.infura.io/v3/${process.env.INFURA_TOKEN}`,
+      // accounts: [`0x${process.env.ACCOUNT_KEY}`],
```

Comment:
This left here intentionally?

Reply:
Yes, I thought it may be useful for future deployments. Without `ACCOUNT_KEY` env var defined the CI fails, that's why I commented it out.